src/Makefile.am

```diff
@@ -126,7 +126,7 @@ TESTS =
 BENCHMARKS =
 
 if BUILD_BITCOIND
-  bin_PROGRAMS += zcoind
+  bin_PROGRAMS += zcoind testbip47
```

Comment:
I think better not to create new binary for bip47 testing, good to include it at test_bitcoin binary and use boost testing for it.